pages/common/ghci.md

```diff
@@ -0,0 +1,24 @@
+# ghci
+
+> The Glasgow Haskell Compiler's interactive environment.
+> More information: <https://downloads.haskell.org/ghc/latest/docs/html/users_guide/ghci.html>.
+
+- Start an interactive REPL session:
+
+`ghci`
+
+- Start an interactive REPL session and load the specified Haskell source file:
+
+`ghci {{source_file.hs}}`
+
+- Start an interactive REPL session and enable a language option:
+
+`ghci -X{{language_option}}`
+
+- Start an interactive REPL session and enable almost all compiler warnings:
+
+`ghci -Wall`
+
+- Start an interactive REPL session with the specified search path for finding source files:
+
+`ghci -i{{dir1:...:dirn}}`
```

Comment:
I think this is better if I can use a relative path to a directory:
```suggestion
`ghci -i{{path/to/directory1}}:{{path/to/directory2}}`
```
Or if the directories should be in the local path:
```suggestion
`ghci -i{{directory1}}:{{directory2}}`
```

Reply:
Relative paths can indeed be used. I will incorporate your suggestion.